pom.xml

```diff
@@ -43,8 +43,9 @@
         <connection>scm:git:https://github.com/PhoenicisOrg/phoenicis.git</connection>
     </scm>
     <properties>
-        <phoenicis.version>5.0-SNAPSHOT</phoenicis.version>
-        <project.settings>${project.basedir}/settings</project.settings>
+        <phoenicis.version>${project.version}</phoenicis.version>
```

Comment:
Does it make sense to have `phoenicis.version`?

Reply:
It is just for backward compatiblity. We shall remove it

Reply:
Is it used anywhere? Can you either open an issue to remove it or remove it in this pr if it's unused?

Reply:
I think it is used for cross dependencies. I'll open a issue